crates/cli/docs/language_guide.md

```diff
@@ -217,8 +217,8 @@ check! 1110
 The `debug` keyword allows you to quickly display a value while working on a
 program.
 
-It prints the result of an expression, prefixed with its line number and the
-original expression as a string.
+It prints the result of an expression, prefixed with its line number, the
+original expression as a string, and its @type if it has one.
```

Comment:
I think I'd prefer to leave this information out at this point in the guide, metakeys get introduced much later on. If you want to document the new behavior maybe you could add something to the `@type` section instead?

Reply:
I think I can just remove it